src/components/Newbeer.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react'
+import Navbar from './Navbar'
+import axios from 'axios';
+
+function NewBeer() {
+    const [name, setName] = useState('');
+    const [tagline, setTagline] = useState('');
+    const [description, setDescription] = useState('');
+    const [firstBrewed, setFirstBrewed] = useState('');
+    const [attenuationLevel, setAttenuationLevel] = useState(0);
+    const [contributedBy, setContributedBy] = useState('');
+
+    const handleName = (e) => setName(e.target.value);
+    const handleTagline = (e) => setTagline(e.target.value);
+    const handleDescription = (e) => setDescription(e.target.value);
+    const handleFirstBrewed = (e) => setFirstBrewed(e.target.value);
+    const handleAttenuationLevel = (e) => setAttenuationLevel(e.target.value);
+    const handleContributedBy = (e) => setContributedBy(e.target.value);
+
+    const handleNewBeer = async (e) => {
+        e.preventDefault();
+        try {
+            const body = { name, tagline, description, firstBrewed, attenuationLevel, contributedBy };
```

Comment:
Since the tags that the `API` accepts for the beer are different from the ones that you named, here you need to do something like this:
`const body = { name, tagline, description, first_brewed: firstBrewed, attenuation_level: attenuationLevel, contributed_by: contributedBy }`